packages/gatsby-plugin-manifest/src/pluginOptionsSchema.js

```diff
@@ -0,0 +1,236 @@
+        `The categories member is an array of strings defining the names of categories that the application supposedly belongs to.`
+      ),
+    description: Joi.string()
+      .optional()
+      .description(
+        `The description member is a string in which developers can explain what the application does. `
+      ),
+    dir: Joi.string()
+      .optional()
+      .valid(`auto`, `ltr`, `rtl`)
+      .description(
+        `The base direction in which to display direction-capable members of the manifest.`
+      ),
+    display: Joi.string()
+      .optional()
+      .valid(`fullscreen`, `standalone`, `minimal-ui`, `browser`)
+      .description(
+        `The display member is a string that determines the developers’ preferred display mode for the website`
+      ),
+    iarc_rating_id: Joi.string()
+      .guid()
+      .optional()
+      .description(
+        `The iarc_rating_id member is a string that represents the International Age Rating Coalition (IARC) certification code of the web application.`
+      ),
+    icons: Joi.array()
+      .optional()
+      .items(ManifestImageResource)
+      .description(
+        `The icons member specifies an array of objects representing image files that can serve as application icons for different contexts.`
+      ),
+    lang: Joi.string()
+      .optional()
+      .description(
+        `The lang member is a string containing a single language tag.`
+      ),
+    name: Joi.string()
+      .optional()
+      .description(
+        `The name member is a string that represents the name of the web application as it is usually displayed to the user.`
+      ),
+    orientation: Joi.string()
+      .optional()
+      .valid(
+        `any`,
+        `natural`,
+        `landscape`,
+        `landscape-primary`,
+        `landscape-secondary`,
+        `portrait`,
+        `portrait-primary`,
+        `portrait-secondary`
+      ) // From: https://www.w3.org/TR/screen-orientation/#screenorientation-interface
+      .description(
+        `The orientation member defines the default orientation for all the website's top-level browsing contexts`
+      ),
+    prefer_related_applications: Joi.boolean()
+      .optional()
+      .description(
+        `The prefer_related_applications member is a boolean value that specifies that applications listed in related_applications should be preferred over the web application.`
+      ),
+    related_applications: Joi.array()
+      .optional()
+      .items(ExternalApplicationResource)
+      .description(
+        `The related_applications field is an array of objects specifying native applications that are installable by, or accessible to, the underlying platform.`
+      ),
+    scope: Joi.string()
+      .optional()
+      .description(
+        `The scope member is a string that defines the navigation scope of this web application's application context.`
+      ),
+    screenshots: Joi.array()
+      .optional()
+      .items(ManifestImageResource)
```

Comment:
There's a [supplementary spec](https://w3c.github.io/manifest-app-info/) (I know, not confusing at all) that contains extra members. The two relevant ones are platform and label. I suggest maybe we at least add label to screenshots because "For accessibility, authors are encouraged to provide a label for each screenshot."

Reply:
Good find. I'll get these added.